lnrpc/walletrpc/walletkit_server.go

```diff
@@ -811,12 +811,26 @@ func (w *WalletKit) SendOutputs(ctx context.Context,
 		return nil, err
 	}
 
+	// Parse coin selection strategy.
+	var coinSelectionStrategy base.CoinSelectionStrategy
```

Comment:
We'll need a general parsing function that parses the new enum into the correct type. Maybe we can pass in the globally configured value and it will then decide which one to use.
A good place for that function would be `lnrpc/marshall_utils.go`.